spec/helpers/category_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the CategoryHelper. For example:
+#
+# describe CategoryHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe CategoryHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I think you can delete all the helper spec files (i.e. view helper specs), since your Helper modules don't actually have any methods in them. If you write some view helpers at some stage, then you can generate the files needed.